app/views/users/index.html.erb

```diff
@@ -1,5 +1,20 @@
-<html>
-  <body>
-    <h1>List of Users</h1>
-  </body>
-</html>
\ No newline at end of file
+
+<div class='container '>
+  <h1 class=''>List of Users</h1>
+  <% @users.each do |user| %>
+    <%= link_to user_path(user.id), class: 'nav-link' do %>
+      <div class='row g-0 border rounded overflow-hidden flex-md-row gap-4 mb-4 shadow-sm h-md-250 position-relative'>
+        <div class='col-auto d-none d-lg-block'>
+          <%= image_tag(user.photo) %>
+        </div>
+        <div class='col p-4 d-flex flex-column position-static'>
+          <h2 class='card-title'><%= user.name %></h2>
+          <div class='d-flex mt-4 justify-content-end'>
+              <p class= 'text-right align-right justify-content-end'> Number of Posts: <%= user.posts_counter %></p>
+          </div>
+        </div>
+      </div>
+      <br/>
+    <% end %>
```

Comment:
- [ ] [optional} @Cosmosha Great work adding the users' page, however, it would be nice to use partial views instead for each loop.

 ```ruby
   # app/views/users/index.html.erb

    <% @users.each do |user| %>
     # HTML Code Here displaying user.photo, user.name and user.posts_counter
    <% end %>
```
 ```ruby
  # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```